### PR TITLE
updated mutation docs to v7 DOC-411

### PR DESCRIPTION
**Pull request by urikirsh**
Jira ticket: https://certora.atlassian.net/browse/DOC-411
Link to generated documentation: https://certora-certora-prover-documentation--230.com.readthedocs.build/en/230/docs/gambit/mutation-verifier.html

### CODE REVIEW

File: docs/gambit/mutation-verifier.md [hunk 1]
```diff
@@ -272,11 +269,39 @@ At the top bar of the report there are different coverage metrics.
...
+| `--debug`                             | Show additional logging information during execution                                                                  |
 
+## Troubleshooting
+
+At the moment, there are a few ways in which `certoraMutate` can fail. 
+Here are some suggestions on how to troubleshoot when that happens. 
+We are actively working on mitigating them.
+
+- There are currently no official Gambit binaries for Linux ARM or Windows. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
+
```
Comment (chandrakananandi) [inline docs/gambit/mutation-verifier.md:300]: should we also add a bullet here to remove trailing `/` from packages?

**Review by chandrakananandi — CHANGES_REQUESTED** [review]
Main issue is that we should not change the gambit.md file here. You can see it says at the top of that file that it is autogenerated.

File: docs/gambit/mutation-verifier.md (hunk 1 above)
Comment (urikirsh) [reply docs/gambit/mutation-verifier.md:300]: done

**Comment by urikirsh** [comment]
I am merging this PR since we release the new version tomorrow, and we want to test the links to some of the new anchors in this PR. Chandra told me to merge this PR in Slack despite the change to `gambit.md`

**Pull request merged**